app/src/main/java/com/jerboa/MainActivity.kt

```diff
@@ -362,6 +362,7 @@ class MainActivity : AppCompatActivity() {
                             blurNSFW = appSettings.blurNSFW.toEnum(),
                             showPostLinkPreviews = appSettings.showPostLinkPreviews,
                             drawerState = drawerState,
+                            onBack = appState::popBackStack,
```

Comment:
Thanks, but can you remove the default `null` from it. We have noticed these defaults have been footguns. They caused issues like this.

Reply:
Not sure what you mean by that, sorry. There is no `null` involved.

Reply:
Unless you mean from the constructor of PersonProfileScreen, but I didn't write that. Could remove it of course.

Reply:
Yes, thats what I mean

Reply:
Removed it from both `onBack` and `padding`, and made them null where they were that by default before.